r2_bucket.go

```diff
@@ -40,6 +42,15 @@ type ListR2BucketsParams struct {
 	Cursor     string `url:"cursor,omitempty"`
 }
 
+type CreateR2BucketParameters struct {
+	Name string `json:"name,omitempty"`
```

Comment:
FWIW you can also pass in a `locationHint` json parameter in here that accepts these values: https://developers.cloudflare.com/r2/buckets/data-location/#available-hints :D

Reply:
Good to know!